src/components/Home/Home.tsx

```diff
@@ -10,7 +10,7 @@ const Home: React.SFC = () => {
         <div className={styles.introContainer}>
           <h1 className={styles.headerText}>Chinatown needs your help</h1>
           <p className={styles.introText}>
-            Help local businesses survive the COVID-19 pandemic and digitize
+            Support local Chinatown businesses during the COVID-19 pandemic and help them digitize
             them the future
```

Comment:
I'm still confused by this sentence

Support local Chinatown businesses during the COVID-19 pandemic and help them digitize them the future them the future?